### ISSUE TITLE
funasr的seaco_paraformer模型训练自己的数据集，经过多次epoch后，更新的准确率一直为0

### ISSUE THREAD

**Issue by gitGoGoGo123** [modelscope/FunASR#2597]
问题：funasr的seaco_paraformer模型训练自己的数据集，是由scp和txt格式数据生成的jsonl格式数据集，经过多次epoch后，loss会逐渐下降，但是更新的准确率一直为0

1.cmd命令
set CUDA_VISIBLE_DEVICES=0 && python train.py --config-name "C:/Users/Desktop/FunASR-main/funasr/bin/config_finetune.yaml" +model="C:/Users/Desktop/FunASR-main/funasr/bin/weight_model/speech_seaco_paraformer_large_asr_nat-zh-cn-16k-common-vocab8404-pytorch" +output_dir="../finetune_models" +train_data_set_list="C:\Users\Desktop\FunASR-main\data\finetune_data_list\train_val\audio_finetune_train_val_datasets1.jsonl" +valid_data_set_list="C:\Users\Desktop\FunASR-main\data\finetune_data_list\train_val\audio_finetune_train_val_datasets2.jsonl" +dataset_conf.hotword_list="C:/Users/Desktop/FunASR-main/funasr/bin/weight_model/speech_seaco_paraformer_large_asr_nat-zh-cn-16k-common-vocab8404-pytorch/hotwords.txt"

2.训练模型yaml配置文件
# network architecture
model: SeacoParaformer
model_conf:
    ctc_weight: 0.0
    lsm_weight: 0.1
    length_normalized_loss: true
    predictor_weight: 1.0
    predictor_bias: 1
    sampling_ratio: 0.75
    inner_dim: 512
    bias_encoder_type: lstm
    bias_encoder_bid: false
    seaco_lsm_weight: 0.3
    seaco_length_normal: true
    train_decoder: true
    NO_BIAS: 8377
    seaco_layers: decoder

# encoder
encoder: SANMEncoder
encoder_conf:
    output_size: 512
    attention_heads: 4
    linear_units: 2048
    num_blocks: 50
    dropout_rate: 0.1
    positional_dropout_rate: 0.1
    attention_dropout_rate: 0.1
    input_layer: pe
    pos_enc_class: SinusoidalPositionEncoder
    normalize_before: true
    kernel_size: 11
    sanm_shfit: 0
    selfattention_layer_type: sanm

# decoder
decoder: ParaformerSANMDecoder
decoder_conf:
    attention_heads: 4
    linear_units: 2048
    num_blocks: 16
    dropout_rate: 0.1
    positional_dropout_rate: 0.1
    self_attention_dropout_rate: 0.1
    src_attention_dropout_rate: 0.1
    att_layer_num: 16
    kernel_size: 11
    sanm_shfit: 0

# seaco decoder
seaco_decoder: ParaformerSANMDecoder
seaco_decoder_conf:
    attention_heads: 4
    linear_units: 1024
    num_blocks: 4
    dropout_rate: 0.1
    positional_dropout_rate: 0.1
    self_attention_dropout_rate: 0.1
    src_attention_dropout_rate: 0.1
    kernel_size: 21
    sanm_shfit: 0
    use_output_layer: false
    wo_input_layer: true

predictor: CifPredictorV3
predictor_conf:
    idim: 512
    threshold: 1.0
    l_order: 1
    r_order: 1
    tail_threshold: 0.45
    smooth_factor2: 0.25
    noise_threshold2: 0.01
    upsample_times: 3
    use_cif1_cnn: false
    upsample_type: cnn_blstm

# frontend related
frontend: WavFrontend
frontend_conf:
    fs: 16000
    window: hamming
    n_mels: 80
    frame_length: 25
    frame_shift: 10
    lfr_m: 7
    lfr_n: 6
    dither: 0.0

specaug: SpecAugLFR
specaug_conf:
    apply_time_warp: false
    time_warp_window: 5
    time_warp_mode: bicubic
    apply_freq_mask: true
    freq_mask_width_range:
    - 0
    - 30
    lfr_rate: 6
    num_freq_mask: 1
    apply_time_mask: true
    time_mask_width_range:
    - 0
    - 12
    num_time_mask: 1

train_conf:
  accum_grad: 1
  grad_clip: 5
  max_epoch: 620
  val_scheduler_criterion:
      - valid
      - acc
  best_model_criterion:
  -   - valid
      - acc
      - max
  keep_nbest_models: 10
  log_interval: 50
  unused_parameters: true

optim: adam
optim_conf:
   lr: 0.0001
scheduler: warmuplr
scheduler_conf:
   warmup_steps: 30000

dataset: AudioDatasetHotword
dataset_conf:
    seaco_id: 8377
    index_ds: IndexDSJsonl
    batch_sampler: DynamicBatchLocalShuffleSampler
    batch_type: example # example or length
    batch_size: 8 # if batch_type is example, batch_size is the numbers of samples; if length, batch_size is source_token_len+target_token_len;
    max_token_length: 2048 # filter samples if source_token_len+target_token_len > max_token_length,
    buffer_size: 500
    shuffle: True
    num_workers: 0
    hotword_file: C:/xxxx/hotwords.txt
    hotword_weight: 4.0
tokenizer: CharTokenizer
tokenizer_conf:
  unk_symbol: <unk>
  split_with_space: true

ctc_conf:
    dropout_rate: 0.0
    ctc_type: builtin
    reduce: true
    ignore_nan_grad: true
normalize: null

3.环境
torch                   2.7.1+cu128                                                                                                                                                
torchaudio          2.7.1+cu128                                                                           
torchvision          0.22.1+cu128 
modelscope        1.27.1  
funasr                  1.2.6

**Comment by CryRobot** [comment]
同样问题，你解决了吗

**Comment by laoxing797373** [comment]
一样的问题，有人解决了吗
